src/main/scala/diplomacy/Resources.scala

```diff
@@ -361,10 +371,34 @@ trait BindingScope
     ResourceMap(SortedMap("/" -> tree))
   }
 
+  /***
+    * Find the device's ResourceBindings in the ResourceBindingsMap
+    * @param dev
+    * @param rsb
+    */
+  def getResourceBindings(dev: Device, rsb: ResourceBindingsMap): Option[ResourceBindings] = {
+    if (rsb.map.contains(dev)) {
+      Some(rsb.map.lift(dev).getOrElse(ResourceBindings()))
+    }
+    else {
+      None
+    }
+  }
```

Comment:
Is this method being used anymore? I don't see it being called anywhere.

Reply:
removed